ods_ci/run_robot_test.sh

```diff
@@ -287,10 +287,17 @@ if command -v yq &> /dev/null
         echo "we did not find yq, so not trying the oc login"
 fi
 
+# Activating the virtual environment
+# python3 -c help you to understand what is going on with environments
```

Comment:
You can just run `poetry env list` and `poetry env info` to get all the information you need